span-reporter/src/main/java/io/opentracing/contrib/reporter/SpanBuilderR.java

```diff
@@ -69,7 +72,14 @@ String findSpanId(SpanContext context) {
     @Override
     public Tracer.SpanBuilder addReference(String s, SpanContext spanContext) {
         wrapped.addReference(s, spanContext);
-        references.put(s, findSpanId(spanContext));
+        String referenceId;
+        if (spanContext != null) {
+            referenceId = spanContext.toSpanId();
+        }
+        else {
+            referenceId = findSpanId(spanContext);
+        }
```

Comment:
regarding the impl of findSpanId, `referenceId` is always `""` in this case

Reply:
> Thanks for the contribution, I need to go back (refresh) into the API of Opentracing (like `spanContext.toSpanId();`) and into the code to complete my review. FYI, I'll not be able to publish a new release (I don't have the permissions to publish)

@davidB - Thanks for your reply. Does that mean there wont be any releases/updates on this library going forward?